content/content-strategy/manage-content-requests/plan-content/content-planning-workshop/example.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: text/callout
+---
+
+### [2]Example acceptance criteria
+
+### [1]User story
```

Comment:
Ignore the callout styling for now. 
I've noticed that there are two heading 3s but the 'User story' one is _displaying_ as a heading 4. Are we going with heading 4s or not?

Reply:
@shad0wMe0w the two ### [1]s are a subset of ### [2] used within an example component

Reply:
Trisha - fixed.